### PR TITLE
[full-ci] [test-only] enable auto-accepting in e2e and UI test

### PR DESCRIPTION
**Pull request by ScharfViktor**
part of https://github.com/owncloud/ocis/issues/7241

I enabled `FRONTEND_AUTO_ACCEPT_SHARES: true` by defaultt

e2e test:  first commit 
- [x] added api step to disable auto-accepting
- [x] deleted `accept share` steps. some tests use UI accepting step to check accepting resources
- [x] deleted step `accepts the following share using API`

ui test:  second commit
- [x] deleted `accept share` steps.
- [x]  deleted some skipped test. related https://github.com/owncloud/web/issues/6896

found bugs: 
- https://github.com/owncloud/web/issues/9776
- backend:  how enabled auto-accepting again?  - double values in the values-list POST `https://host.docker.internal:9200/api/v0/settings/values-list` 
- backend: when autoaccepting is enable -> `new` dav doesn't return any resources more details here https://github.com/owncloud/ocis/issues/7440 - that is the reason failed UI tests

TODO
- [ ] delete steps from middleware `user "Alice" has accepted the share "Shares/lorem.txt" offered by user "Brian" in the server` and `And user "Carol" accepts the share "Shares/simple-folder" offered by user "Alice" using the sharing API in the server`
- [ ] the setting "shareapi_auto_accept_share" of app "core" has been set to "no" in the server

### CODE REVIEW

**Comment by ScharfViktor** [comment]
Failed steps: 
- `And user "Alice" has shared folder "/Shares/simple-folder" with user "Carol" in the server`
- `And the content of file "Shares/randomfile.txt" for user "Brian" should be "Carol file" in the server`
- `And user "Alice" has shared folder "/Shares/simple-folder" with user "Carol" with "read, delete" permissions in the server`


middleware uses `new` dav path where some endpoints don't work like here: https://github.com/owncloud/ocis/issues/7440

web uses space dav so webUI steps work. 

I see three ways to get a "green" CI:
- wait for the fix https://github.com/owncloud/ocis/issues/7440
- change the middleware to use space dav (check first: oc10 should not use these steps)
- skip these steps temporarily

**Review by saw-jan — COMMENTED** [review]
Generally, looks good :+1:

**Comment by saw-jan** [comment]
>- wait for the fix

Would be great if dev team could make a fix (I would vote for this one if the fix can be quick)

>- change the middleware to use space dav (check first: oc10 should not use these steps)

We don't know how this change could affect other test scenarios

>- skip these steps temporarily

We could put those failing tests in expected failure list and we would know if the issue is fixed when we do ocis bump - tests should pass. (I would go for this as second option)

**Comment by SwikritiT** [comment]
> > * wait for the fix
> 
> Would be great if dev team could make a fix (I would vote for this one if the fix can be quick)
> 
> > * change the middleware to use space dav (check first: oc10 should not use these steps)
> 
> We don't know how this change could affect other test scenarios
> 
> > * skip these steps temporarily
> 
> We could put those failing tests in expected failure list and we would know if the issue is fixed when we do ocis bump - tests should pass. (I would go for this as second option)

If the fix won't be soon then I would also vote for keeping the tests in expected to fail and merging this PR

**Comment by ScharfViktor** [comment]
There's a fix coming soon https://github.com/cs3org/reva/pull/4244 so I'm waiting.

**Comment by ScharfViktor** [comment]
CI is green. could you have look

**Review by kobergj — APPROVED** [review]

File: tests/e2e/cucumber/features/smoke/link.feature [hunk 1]
```diff
@@ -98,22 +98,16 @@ Feature: link
...
-    When "Brian" opens the public link "imageLink"
-    Then for "Brian" file "testavatar.jpg" should be selected
+    # When "Brian" opens the public link "textLink"
+    # Then for "Brian" file "shareToBrian.txt" should be selected
+    # When "Brian" opens the public link "markdownLink"
+    # Then for "Brian" file "shareToBrian.md" should be selected
+    # When "Brian" opens the public link "pdfLink"
+    # Then for "Brian" file "simple.pdf" should be selected
+    # When "Brian" opens the public link "imageLink"
+    # Then for "Brian" file "testavatar.jpg" should be selected
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/link.feature:112]: commented out?

File: tests/e2e/support/api/userSettings/settings.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import join from 'join-path'
+
+import { checkResponseStatus, request } from '../http'
+import { User } from '../../types'
+
+export const disableAutoAccepting = async ({ user }: { user: User }): Promise<void> => {
```
Comment (saw-jan) [inline tests/e2e/support/api/userSettings/settings.ts:6]:
```suggestion
export const disableAutoAcceptShare = async ({ user }: { user: User }): Promise<void> => {
```

File: tests/e2e/support/api/userSettings/settings.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      boolValue: false
+    }
+  })
+  const response = await request({
+    method: 'POST',
+    path: join('api', 'v0', 'settings', 'values-save'),
+    body: body,
+    user: user
+  })
+  checkResponseStatus(response, 'Failed while disabling auto accept')
```
Comment (saw-jan) [inline tests/e2e/support/api/userSettings/settings.ts:24]:
```suggestion
  checkResponseStatus(response, 'Failed while disabling auto-accept share')
```

File: tests/e2e/cucumber/features/smoke/link.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/features/smoke/link.feature:112]: done, thanks  :thumbsup:

**Comment by SwikritiT** [comment]
File changes shows `141` is that correct?

**Comment by ScharfViktor** [comment]
> File changed shows `141 files changes` is that correct?

no, that was a bad rebase. Now is good

**Review by saw-jan — APPROVED** [review]
LGTM :+1: :sparkles:

**Pull request merged**